stories/Select/Select.tsx

```diff
@@ -0,0 +1,134 @@
+import { Fragment, useEffect, useState } from "react";
+import { Listbox, Transition } from "@headlessui/react";
+import { CheckIcon, SelectorIcon } from "@heroicons/react/solid";
+
+interface SelectValue {
+  name: string;
+  value: string;
+  isSelected?: boolean;
+  isDisabled?: boolean;
+}
+
+interface SelectProps {
+  /**
+   * Optional description of the select input
+   */
+  description?: string;
+  /**
+   * Optional error message of the select input
+   */
+  errorMessage?: string;
+  /**
+   * Whether the input is disabled
+   */
+  isInputDisabled?: boolean;
+  /**
+   * Whether the label should be visible
+   */
+  isLabelVisible?: boolean;
+  /**
+   * Descriptive label of the select input
+   */
+  label: string;
+  /**
+   * Optional placeholder of the select input
+   */
+  placeholder?: string;
+  /**
+   * Optional placeholder of the select input
+   */
+  values: SelectValue[];
+}
+
+const Select = ({
+  description,
+  errorMessage,
+  isInputDisabled,
+  isLabelVisible,
+  label,
+  placeholder,
+  values,
+}: SelectProps) => {
+  const [selected, setSelected] = useState(values[0]);
+
+  useEffect(() => {
+    const defaultValue = values.find((value) => value.isSelected);
+    if (defaultValue) {
+      setSelected(defaultValue);
+    }
+  }, []);
+
+  return (
+    <div className="fixed top-16 w-72">
+      <Listbox
+        disabled={isInputDisabled}
+        onChange={setSelected}
+        value={selected}
+      >
+        {/* Creo que esto no resuelve ok el tema de que no se renderice si isLabelVisible
+    es false, pero no encontré otra manera mejor de hacerlo. */}
+        {isLabelVisible ? (
+          <Listbox.Label>{label}</Listbox.Label>
+        ) : (
+          <Listbox.Label aria-label={label}></Listbox.Label>
+        )}
```

Comment:
Tambien podrias hacer algo como:

```suggestion
        <Listbox.Label className={isLabelVisible ? "" : "sr-only"}>
          {label}
        </Listbox.Label>
```
Tailwind tiene la clase `sr-only` para ocultar visualmente un elemento pero dejandolo en el DOM
More info: https://tailwindcss.com/docs/screen-readers

Reply:
Ahh, esto resuelve todo creo, gracias!